platforms/unix/vm/sqUnixExternalPrims.c

```diff
@@ -202,11 +203,7 @@ void *ioLoadModule(char *pluginName)
 
   /* try dlopen()ing LIBRARY_PREFIX<name>LIBRARY_SUFFIX searching only the default locations modulo LD_LIBRARY_PATH et al */
 
-# if defined(HAVE_SNPRINTF)
```

Comment:
hi Gerardo,  
A good initiative, but these HAVE_SNPRINTF checks indicate some platforms don't support snprintf().  However this code is from 2010 and maybe there is broader support now. But in particular MSVC didn't support snprintf() until 2015. I do see Nicolas reports successful builds using MSVC2015 - so I guess that would become a the minimum requirement for native Windows.  

Any platforms missing snprintf() might pick up a portable version [1] from instead #defining snprintf() as sprintf()...
[1] https://github.com/search?utf8=%E2%9C%93&q=snprintf